Scripts/update-translations.rb

```diff
@@ -86,20 +90,35 @@ def copy_comment(f, trans_strings, value)
   langs = ALL_LANGS
 end
 
+fallback_lang_dir = File.join('Simplenote', "#{FALLBACK_LANG}.lproj")
+
 langs.each do |code,local|
   lang_dir = File.join('Simplenote', "#{local}.lproj")
   puts "Updating #{code}"
   system "mkdir -p #{lang_dir}"
   
-  # Backup the current file
+  # Backup the current file(s)
   system "if [ -e #{lang_dir}/Localizable.strings ]; then cp #{lang_dir}/Localizable.strings #{lang_dir}/Localizable.strings.bak; fi"
+  system "if [ -e #{lang_dir}/MainMenu.strings ]; then cp #{lang_dir}/MainMenu.strings #{lang_dir}/MainMenu.strings.bak; fi"
```

Comment:
Suggestion for the future, this could also be done via the Ruby `File.exists?` and `FileUtils.copy` APIs.

Reply:
Thank you sir!!! taking note, fixing in a follow up.